src/main/webapp/src/app/schedule-page/schedule-page.component.html

```diff
@@ -0,0 +1,81 @@
+<div class='flex-container'>
+  <div style='flex-grow: 2'>
+    <div class='card-padding'>
+      <mat-card>
+        <h2> Peapod Garden </h2>
+        <h4> Caroline | New York City, NY </h4>
+        <button md-raised-button class='button-style'> Go back </button>
+      </mat-card>
+    </div>
+
+    <div class='card-padding'>
+      <mat-card>
+        <mat-accordion>
+          <mat-expansion-panel hideToggle>
+            <mat-expansion-panel-header>
+              <mat-panel-title> About the garden </mat-panel-title>
+            </mat-expansion-panel-header>
+              <p>Growing all the green things that look like peas.</p>
+          </mat-expansion-panel>
+
+          <mat-expansion-panel hideToggle>
+            <mat-expansion-panel-header>
+              <mat-panel-title> Plants </mat-panel-title>
+            </mat-expansion-panel-header>
+              <ul>
+                <li>Pea plant</li>
+                <li>Pears</li>
+              </ul>
+          </mat-expansion-panel>
+
+          <mat-expansion-panel hideToggle>
+            <mat-expansion-panel-header>
+              <mat-panel-title> Members </mat-panel-title>
+            </mat-expansion-panel-header>
+              <ul>
+                <li>Caroline (owner)</li>
+                <li>Jake</li>
+                <li>Kayla</li>
+                <li>Cody</li>
+                <li>Stephanie</li>
+              </ul>
+          </mat-expansion-panel>
+        </mat-accordion>
+      </mat-card>
+    </div>
+  </div>
+
+  <div style='flex-grow: 4'>
```

Comment:
nit: why have this `flex-grow` property as an inline style vs external css

Reply:
I was testing the flex grow property but I just saw your demo so I'll be making some changes